bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,695 @@
+		<item-type>String</item-type>
+		<label>Device ID (from Data)</label>
+		<state readOnly="true"></state>
+		</channel-type>
+	-->
+	<channel-type id="status" advanced="false">
+		<item-type>String</item-type>
+		<label>Status of Sensors</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="typps" advanced="false">
+		<item-type>Number:Length</item-type>
+		<label>Average size of Fine Dust</label>
+		<state readOnly="true" pattern="%.2f μm"></state>
+	</channel-type>
+
+	<!-- <channel-type id="bat" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Battery State</label>
+		<state readOnly="true"></state>
+		</channel-type>
+	-->
+	<channel-type id="cnt0_3" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fine Dust >0,3 μm</label>
+		<state readOnly="true" pattern="%.0f"></state>
+	</channel-type>
+
+	<channel-type id="cnt0_5" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fine Dust >0,5 μm</label>
+		<state readOnly="true" pattern="%.0f"></state>
+	</channel-type>
+
+	<channel-type id="cnt1" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fine Dust >1 μm</label>
+		<state readOnly="true" pattern="%.0f"></state>
+	</channel-type>
+
+	<channel-type id="cnt2_5" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fine Dust >2,5 μm</label>
+		<state readOnly="true" pattern="%.0f"></state>
+	</channel-type>
+
+	<channel-type id="cnt5" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fine Dust >5 μm</label>
+		<state readOnly="true" pattern="%.0f"></state>
+	</channel-type>
+
+	<channel-type id="cnt10" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fine Dust >10 μm</label>
+		<state readOnly="true" pattern="%.0f"></state>
+	</channel-type>
+
+	<channel-type id="co" advanced="false">
+		<item-type>Number</item-type>
+		<label>CO Concentration</label>
+		<state readOnly="true" pattern="%.0f mg/m³"></state>
+	</channel-type>
+
+	<channel-type id="co2" advanced="false">
+		<item-type>Number</item-type>
+		<label>CO2 Concentration</label>
+		<state readOnly="true" pattern="%.0f ppm"></state>
+	</channel-type>
+
+	<channel-type id="dco2dt" advanced="false">
+		<item-type>Number</item-type>
+		<label>Change of CO2 Concentr.</label>
+		<state readOnly="true" pattern="%.2f ppm/s"></state>
+	</channel-type>
+
+	<channel-type id="dhdt" advanced="false">
+		<item-type>Number</item-type>
+		<label>Change of Humidity</label>
+		<state readOnly="true" pattern="%.2f g/m³/s"></state>
+	</channel-type>
+
+	<channel-type id="dewpt" advanced="false">
+		<item-type>Number:Temperature</item-type>
+		<label>Dew Point</label>
+		<state readOnly="true" pattern="%.3f %unit%"></state>
+	</channel-type>
+
+	<channel-type id="door" advanced="false">
+		<item-type>Number</item-type>
+		<label>Door Event (exp)</label>
+		<state readOnly="true"></state>
+	</channel-type>
```

Comment:
Is this rather a trigger Channel? See https://www.openhab.org/docs/developer/bindings/thing-xml.html#trigger-channel-types

Reply:
Thank you for pointing me to the trigger channels. This channel does not 'fire' an event, and I would prefer it to deliver this status about the door (if it works one day; currently it doesn't for me, my air-Q never signals an open door). Trigger channels don't have a status, so I would keep this as a regular channel.

Reply:
OK